var/spack/repos/builtin/packages/rpp/package.py

```diff
@@ -45,9 +45,11 @@ def url_for_version(self, version):
     variant("opencl", default=False, description="Use OPENCL as the backend")
     variant("hip", default=True, description="Use HIP as backend")
     variant("cpu", default=False, description="Use CPU as backend")
+    variant("add_tests", default=False, description="add utilities folder")
 
     patch("0001-include-half-openmp-through-spack-package.patch")
     patch("0002-declare-handle-in-header.patch")
+    patch("0003-changes-to-rpp-unit-tests.patch", when="+add_tests")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
comment added